include/mbedtls/md.h

```diff
@@ -79,7 +79,7 @@ typedef enum {
 #endif
 
 /**
- * Opaque struct defined in md_internal.h.
+ * Opaque struct defined in md_wrap.h.
```

Comment:
This is public documentation, but there's no `md_wrap.h` referenced in the public documentation. At least
```suggestion
 * Opaque struct defined in `library/md_wrap.h`.
```

Better, change the text in the Doxygen comment to just say that it's an opaque struct and preferably (but getting out of scope) reference the functions that apply to it (`mbedtls_md_info_from_xxx` to construct one, `mbedtls_md_get_xxx` to access the contents), and add a non-Doxygen comment referencing `library/md_wrap.h`.

This goes for the other places where headers that are now in `library` are still referenced in a comment (Doxygen or not) under `include`.

Reply:
Done.